champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/SendSkills.java

```diff
@@ -47,6 +52,10 @@ public String getDescription() {
 
     @Override
     public void execute(Player player, Client client, String... args) {
+        if(!cooldownManager.use(player, getName(), 1000d * 15d, false, true)){
```

Comment:
This is in seconds btw, you sure you want a 15,000 second cooldown?

Reply:
I fail to see the issue